scripts/update_schemas.py

```diff
@@ -1,21 +1,25 @@
 #!/usr/bin/env python3
 
 import os
+import re
 import requests
 
 from json import dumps
 
 DIRECTORY = os.path.dirname(__file__)
+RE_YAML = re.compile(r'\.ya?ml$')
```

Comment:
I am ok with the RE_YAML regex. Just wanted to ask why you didn't wrote :D `RE_JSON = re.compile(r'\.json$')` and later instaead of asking ` not RE_YAML.search(pattern)` we can just ask `RE_JSON.search(pattern`

Reply:
Because there are various file patterns that don't end with .json that should still be supported. For example `.bowerrc`. That will work as long as the user manually associates that file pattern with JSON syntax.